doc/developer/requirements.md

```diff
@@ -0,0 +1,129 @@
+
+You may also need at some point to restart the _Command Prompt_, the whole
+system, or refresh your environment variables using the`refreshenv` command.
+
+Then install the Windows build tools from an admin console as documented in the
+[Microsoft's Node.js guidelines](https://github.com/Microsoft/nodejs-guidelines/blob/master/windows-environment.md#environment-setup-and-configuration):
+
+    npm install -g windows-build-tools
+
+You may still need to manually add the python installation directory to your
+`PATH`.
+To do so, search for _PATH_ from the start menu and select
+_Edit environment variables for your account_.
+Then edit the `PATH` user variable (not the system one) and append the
+following to the end (assuming `...` is the current text):
+
+    `...;%USERPROFILE%\.windows-build-tools\python27`
+
+To set up Docker, in case you hardware is old or you don't own a Windows Pro
+license, you may encounter the same issue as on old macOS hardware (see below).
+_(please feel free to improve this section)_
+
+### macOS
+
+You need a Mac to build / test for macOS.
+
+The easiest way to set up the environment is to install and use
+[Homebrew](https://brew.sh/):
+
+    brew install git
+    brew install node
+    brew install yarn
+
+To install Docker, first check that your mac has kernel hypervisor support:
+
+    sysctl -n kern.hv_support
+
+If the output is `1`, then you can install the latest Docker version:
+
+    brew cask install docker
+
+Otherwise you'll have to install a Docker VM and set up your shell environment
+so the various docker commands know how to use it (you can omit the `default`
+parameters below):
+
+    brew cask install docker-toolbox
+    docker-machine create default
+    eval $(docker-machine env default)
+
+### Fedora 27
+
+    sudo dnf install docker gcc-c++ git nodejs
+    cd /etc/yum.repos.d/
+    sudo wget https://dl.yarnpkg.com/rpm/yarn.repo
+    cd -
+    sudo dnf install yarn
+
+### Ubuntu
+
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    sudo add-apt-repository \
+     "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    sudo apt update
+    sudo apt install git nodejs build-essential gcc transifex-client icnsutils graphicsmagick python-pip docker-ce yarn
+    sudo pip install docker-compose
+    sudo groupadd docker
+    sudo usermod -aG docker $USER
+
+## Set up a Cozy stack
+
+If you don't already have a [running Cozy stack](https://github.com/cozy/cozy-stack/blob/master/docs/INSTALL.md), the easiest way to get started is to use Docker:
+
+    cd cli
+    docker-compose up
+    yarn bootstrap
+
+Otherwise, please refer to the
+[../../cli/dev/docker/bootstrap-cozy-desktop.sh]()
+script to create the appropriate instances, token, etc...
+
+## Cozy Stack
```

Comment:
This section is redundant with the previous one and doesn't look like a requirement.
What about moving it to `doc/developer/docker.md`?